app/src/main/java/com/firebase/uidemo/database/ChatActivity.java

```diff
@@ -93,11 +94,7 @@ public void onComplete(DatabaseError databaseError, DatabaseReference reference)
         });
 
         mMessages = (RecyclerView) findViewById(R.id.messagesList);
-
         mManager = new LinearLayoutManager(this);
-        mManager.setReverseLayout(false);
-
-        mMessages.setHasFixedSize(false);
```

Comment:
I removed this change, but I am curious, why are we setting default values and adding seemingly unnecessary code?